pkg/common/telemetry/metrics.go

```diff
@@ -48,7 +48,8 @@ var _ Metrics = (*MetricsImpl)(nil)
 // NewMetrics returns a Metric implementation
 func NewMetrics(c *MetricsConfig) *MetricsImpl {
 	// Always create an in-memory sink
-	interval := 1 * time.Second
+	interval := 1 * time.Minute
+	//interval := 1 * time.Second
```

Comment:
Was this change intentional? IMO 1 second is a reasonable emission frequency for runtime-related statistics

Reply:
It was not. Will revert.